src/pages/api/_swagger.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import DocumentTitle from 'react-document-title'
+
+import {Tabs} from 'antd';
+import {StaticQuery, graphql} from 'gatsby'
+
+import Link from 'components/Link'
+import Swagger from '../../components/Swagger'
+
+import './swagger.css'
+
+import docs from './fleerp.json'
+
+export class SwaggerWrapper extends React.Component {
+  static propTypes = {}
+
+  render() {
+    const paths = {
+      'pois': [],
+      'users': [],
+      'zones': [],
+      'alarms': [],
+      'groups': [],
+      'search': [],
+      'drivers': [],
+      'tobjects': [],
+      'settings': [],
+      'ecodriving': [],
+      'invitations': [],
+      'userConfigs': [],
+      'vehicleTypes': [],
+      'deviceVendors': [],
+      'mobileCarriers': [],
+      'configTemplates': [],
+      'maintenanceTasks': [],
+      'externalServices': []
+    }
+
+
+    for (let [key, value] of Object.entries(docs.paths)) {
```

Comment:
For Politis we have to add similar code which doesn't sound right. It is better this code to relay only on the content that comes from json/yaml definition.